M2/Macaulay2/packages/AbstractSimplicialComplexes.m2

```diff
@@ -653,17 +670,20 @@ doc ///
 	     L = randomAbstractSimplicialComplex(6,3)
 	  Text
 	     Create the random complex $Y_d(n,m)$ which has vertex set
-             $[n]$ and $(d − 1)$-skeleton, and has $m$ $d$-dimensional faces,
+             $[n]$ and complete $(d − 1)$-skeleton, and has exactly m d-dimensional faces,
              chosen at random from all $\binom{\binom{n}{d+1}{m}$ possibilities.
```

Comment:
The TeX for this binomial coefficient isn't rendering correct -- I think you need an extra closing curly brace